diff --git a/src/util/ICM_20948_C.h b/src/util/ICM_20948_C.h
index 315c6a6..2d56efa 100644
--- a/src/util/ICM_20948_C.h
+++ b/src/util/ICM_20948_C.h
@@ -31,7 +31,7 @@ extern int memcmp(const void *, const void *, size_t); // Avoid compiler warning
 
 // Define if the DMP will be supported
 // Note: you must have 14290/14301 Bytes of program memory available to store the DMP firmware!
-//#define ICM_20948_USE_DMP // Uncomment this line to enable DMP support. You can of course use ICM_20948_USE_DMP as a compiler flag too
+#define ICM_20948_USE_DMP // Uncomment this line to enable DMP support. You can of course use ICM_20948_USE_DMP as a compiler flag too
 
 // There are two versions of the InvenSense DMP firmware for the ICM20948 - with slightly different sizes
 #define DMP_CODE_SIZE 14301 /* eMD-SmartMotion-ICM20948-1.1.0-MP */
